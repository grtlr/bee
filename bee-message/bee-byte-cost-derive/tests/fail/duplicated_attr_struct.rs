// Copyright 2021 IOTA Stiftung
// SPDX-License-Identifier: Apache-2.0

#![allow(unused_imports, unreachable_patterns)]

use bee_byte_cost::ByteCost;

#[derive(Default, ByteCost)]
struct SingleField {
    #[byte_cost(key)]
    #[byte_cost(data)]
    a: u32,
}

fn main() {}
