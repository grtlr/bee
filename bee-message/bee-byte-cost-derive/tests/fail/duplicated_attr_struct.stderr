error: there can be at most one `#[byte_cost(...)]` attribute per field
  --> tests/fail/duplicated_attr_struct.rs:10:5
   |
10 |     #[byte_cost(key)]
   |     ^
